samples/test/datasets.test.js

```diff
@@ -29,7 +29,8 @@ const datasetId = `${GCLOUD_TESTS_PREFIX}_datasets_${uuid.v4()}`.replace(
 
 const bigquery = new BigQuery();
 
-describe('Datasets', () => {
+describe('Datasets', function () {
+  this.retries(3);
```

Comment:
If we want to try 3 times, we actually need to pass `2` as parameter, as the test will run once + number of retries.